src/resources/Subscriptions.ts

```diff
@@ -339,6 +340,20 @@ export class Subscriptions extends CRUDResource {
         return this._list(data, auth, { storeId });
     }
 
+    private _suspend?: DefinedRoute;
+    suspend(storeId: string, id: string, data?: SendData<void>, auth?: AuthParams): Promise<ResponsePayment> {
```

Comment:
I think for suspend / unsuspend we also need the merchantId? 🤔

Reply:
Merchant id is for the private SDK. No need for the public one.